web/sites.css

```diff
@@ -1,195 +1,206 @@
 html {
-    height: 100%;
+  height: 100%;
 }
 
 body {
-    margin: 0;
-    padding: 0;
-    height: 100%;
-    width: 100%;
+  margin: 0;
+  padding: 0;
+  height: 100%;
+  width: 100%;
 }
 
-p,h1,form,button {
```

Comment:
Not sure, this might be just be me, but I think it is easier to read to have all the elements that are changed on a single line instead of them listed one by one.